api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -88,12 +88,21 @@ public Profile getProfile(User user) {
     if (user.getTrainingCompletionTime() != null) {
```

Comment:
question-- will `trainingCompletion` eventually get replaced by `complianceTrainingCompletion`?

Reply:
Yes, not sure what the plan is on sequencing though. I think @gjuggler probably has the latest thinking on that.

Reply:
Sorry for the delayed response here.

Yes, we'll eventually migrate the field name.

We'll want to do the migration (DB migration script, and update Moodle API calls to refer to the new field) before we turn on the access.enableComplianceTraining flag in production. I'll propose we take care of this (and other migrations) in the next sprint.

Reply:
@gjuggler we should talk to Karthik about that ASAP if so, I think we can do it, but we will need to make sure that gets prioritized in.

Reply:
I'll make a note in https://precisionmedicineinitiative.atlassian.net/browse/RW-2331 (Athens production readiness). As it stands we're not quite ready for Moodle to be turned on, so this cleanup is a required part of that ticket. I don't think this reflects any shift in priorities – just that there's still a few steps left before we can just "switch it on"

Reply:
Got it. I can also sync up with Karthik in person on this on Monday, if there's anything we feel is unresolved